dist/templates/rbac-ovnkube-node.yaml.j2

```diff
@@ -118,8 +118,8 @@ metadata:
 rules:
     - apiGroups: [""]
       resources:
-          - nodes/status
-      verbs: [ "get" ]
+          - nodes
```

Comment:
Let's rename the ClusterRole to `node-reader` or similar? I would hate to go with `ovnkube-node-node-reader` :D

Reply:
Hey sorry, to clarify this, do you wan to drop the ovnkube-node prefix? Woudn't that make it confusing to identify the clusterrole? 

Also, do I need to manually change the `helm/ovn-kubernetes/templates/rbac-ovnkube-node.yaml` file, or is there some automation to copy the changes?